README.md

```diff
@@ -28,58 +30,27 @@ Total 134 (delta 54), reused 128 (delta 52)
        "Retrieving" "ring/ring-jetty-adapter/1.4.0/ring-jetty-adapter-1.4.0.jar" "from" "clojars"
        "Retrieving" "ring/ring-servlet/1.4.0/ring-servlet-1.4.0.jar" "from" "clojars"
        Compiling helloworld.web
-       2016-03-22 22:27:02.406:INFO::main: Logging initialized @8351ms
+       2016-06-07 16:05:50.783:INFO::main: Logging initialized @4618ms
 -----> Discovering process types
        Procfile declares types -> web
        Default process types for Clojure (Leiningen 2) -> web
 -----> Compiled slug size is 67M
+Build complete.
+Launching App...
+Done, haptic-duckling:v2 deployed to Deis
 
------> Building Docker image
-remote: Sending build context to Docker daemon 70.12 MB
-Step 0 : FROM deis/slugrunner
-# Executing 3 build triggers
-Trigger 0, RUN mkdir -p /app
-Step 0 : RUN mkdir -p /app
- ---> Running in 476207d4b615
-Trigger 1, WORKDIR /app
-Step 0 : WORKDIR /app
- ---> Running in bcde6c0b43ce
-Trigger 2, ADD slug.tgz /app
-Step 0 : ADD slug.tgz /app
- ---> d8a0a81b9ddc
-Removing intermediate container 476207d4b615
-Removing intermediate container bcde6c0b43ce
-Removing intermediate container aac959a322ba
-Step 1 : ENV GIT_SHA f1de81a9f9bb151b8843c1650baedece547a2429
- ---> Running in d4c4c226e9d7
- ---> 0eaa4fe602f3
-Removing intermediate container d4c4c226e9d7
-Successfully built 0eaa4fe602f3
------> Pushing image to private registry
------> Launching...
-       done, velvet-jerrycan:v2 deployed to Deis
+Use 'deis open' to view this application in your browser
 
-       http://velvet-jerrycan.local3.deisapp.com
+To learn more, use 'deis help' or visit http://deis.io
```

Comment:
Oops--looks like we need to fix this in workflow-cli to point to https://deis.com/. I'll PR that.